addons/membership/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
Few of the py files are having licence stuff while others aren't. Can you make it uniform ?